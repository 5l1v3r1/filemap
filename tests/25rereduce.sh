#!/bin/sh

fm -v map -i '/test/*' "`pwd`/split.sh < sort > sort -m | uniq -c"
#fm ls /reduce/*/*
