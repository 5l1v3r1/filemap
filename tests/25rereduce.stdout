      2 4	5	6
      2 1	2	3
      1 1	3	5
      1 1	2	8
      2 7	8	9
